components/Slider.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as SliderPrimitive from '@radix-ui/react-slider';
+
+const Slider = ({value, min, max, step, label, changeHandler}) => (
```

Comment:
```typescript
// I like doing:
const { label, value, primitiveProps } = { ...props };

<SliderPrimitive.Root 
    className="slider mt-2"
    value={[value]} 
    aria-label={label}
    ...primitiveProps />
```

That's personal preference though - `onValueChange` reads better to me than `changeHandler`.

Reply:
Sounds good. I'll keep that in mind for future PRs.